.zenodo.json

```diff
@@ -22,44 +27,48 @@
       "affiliation": "Helmholtz-Zentrum Dresden-Rossendorf",
       "orcid": "0000-0002-8218-3116"
     },
+    {
+      "name": "Ferragina, Luca",
+      "affiliation": "CERN"
```

Comment:
@Parsifal-2045 I wasn't able to find an ORCID for you. Do you have one?

Reply:
I do now 0009-0004-3148-0315